resource_activity/models/sale.py

```diff
@@ -2,10 +2,15 @@
 # Copyright 2018 Coop IT Easy SCRLfs.
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openerp import api, fields, models
+from openerp import api, fields, models, _
 from openerp.exceptions import UserError
 
 
+class ResCompany(models.Model):
+    _inherit = "res.company"
+    sale_note = fields.Html(string='Default Terms and Conditions', translate=True, sanitize=False)
+
+

```

Comment:
If you don't put it in a dedicated file. It has more sense to put it in sale_note.py from my point of view.

Reply:
Rather in `res_company.py` 🙏

Reply:
Actually, here I'm changing the field type (.Text -> .Html) of the [standard sale_note](https://github.com/odoo/odoo/blob/9.0/addons/sale/sale.py#L14). The client needs the sale_note are in HTML and not in text. Should I create a new field.Html `sale_note_html` instead and not change the existing one ?

Reply:
Yes it's probably a better idea to make another field rather than change the type.

Reply:
I agree changing the type of an existing field of module on which you depends will lead to lot of problems because during an update all the process goes trough all the module following the depedency tree. So updating the original module the field will set back to char then updating yours it will switch back the field to html again loosing the content of the fields. and creating field_name_old,field_name_old2,field_name_old3 and so on in the database.

create a new field instead.

Reply:
new fields `note_html` (`sale.order`) and `sale_note_html` (`res.company`) added